docs/guide/core-concepts.md

```diff
@@ -0,0 +1,427 @@
+because Int.MinValue is the GeoTrellis-default `NoData` value for
+integers */
+assert(normalTile.get(0, 0) == 42)
+assert(normalTile.getDouble(0, 0) == 42.0)
+
+/** Here, the result is less obvious. Under the hood, GeoTrellis is
+inspecting the value to be returned at (0, 0) to see if it matches our
+custom `NoData` policy and, if it matches (it does), return Int.MinValue
+(no matter your underlying type, `get` on a tile will return an `Int`
+and `getDouble` will return a `Double`.
+
+The use of Int.MinValue and Double.NaN is a result of those being the
+GeoTrellis-blessed values for NoData - below, you'll find a chart that
+lists all such values in the rightmost column */
+assert(customTile.get(0, 0) == Int.MinValue)
+assert(customTile.getDouble(0, 0) == Double.NaN)
+```
+
+**Why you should care**
+
+In most programming contexts, it isn't all that useful to think carefully
+about the number of bits necessary to represent the data passed around
+by a program. A program tasked with keeping track of all the birthdays
+in an office or all the accidents on the New Jersey turnpike simply
+doesn't benefit from carefully considering whether the allocation of
+those extra few bits is *really* worth it. The costs for any lack of
+efficiency are more than offset by the savings in development time and
+effort. This insight - that computers have become fast enough for us to
+be forgiven for many of our programming sins - is, by now, truism.  
+
+An exception to this freedom from thinking too hard about
+implementation details is any software that tries, in earnest, to
+provide the tools for reading, writing, and working with large arrays of
+data. Rasters certainly fit the bill. Even relatively modest rasters
+can be made up of millions of underlying cells. Additionally,
+the semantics of a raster imply that each of these cells shares
+an underlying data type. These points - that rasters are made up
+of a great many cells and that they all share a backing
+data type - jointly suggest that a decision regarding the underlying
+data type could have profound consequences. More on these consequences
+[below](#cell-type-performance).  
+
+Compliance with the GeoTIFF standard is another reason that management
+of cell types is important for GeoTrellis. The most common format for
+persisting a raster is the [GeoTIFF](https://trac.osgeo.org/geotiff/).
+A GeoTIFF  is simply an array of data along with some useful tags
+(hence the 'tagged' of 'tagged image file format'). One of these
+tags specifies the size of each cell and how those bytes should be
+interpreted (i.e. whether the data for a byte includes its
+sign - positive or negative - or whether it counts up from 0 - and
+is therefore said to be 'unsigned').  
+
+In addition to keeping track of the memory used by each cell in a `Tile`,
+the cell type is where decisions about which values count as data (and
+which, if any, are treated as `NoData`). A value recognized as `NoData`
```

Comment:
"... is where decisions about which values count as data." needs a verb.  Perhaps "are stored"?